README.md

```diff
@@ -4,9 +4,9 @@ Kinase (pronounced ki·nase) is a pluggable browser extension allowing you to la
 
 ## Concepts
 ### Annotations
-Annotations represent the labels which you’d like to select content for. An annotation is represented by a string identifier (e.g., `products`) and a schema containing the fields and their types that can be mapped to the annotation (e.g., `name`, `description`, `photo`). The schema can optionally specify the `multiple` flag to indicate that multiple groupings of these fields can be mapped to the annotation.
+Annotations represent the labels for which you’d like to select content. An annotation is represented by a string identifier (e.g., `products`) and a schema containing the fields and their types that can be mapped to the annotation (e.g., `name`, `description`, `photo`). The schema can optionally specify the `multiple` flag to indicate that multiple groupings of these fields can be mapped to the annotation.
 
-NB: Currently only `text`, `rich-text`, and `image` fields are supported. Help us out and add more!
+The currently supported field types are `text`, `rich-text`, and `image`. Help us out and add more!
 
```

Comment:
Here is an example description of a collection `products` including a product's schema: